platform/languages.hpp

```diff
@@ -40,6 +40,7 @@ std::array<std::pair<std::string, std::string>, 39> const kLanguageList =
   {"cs", "Čeština"},
   {"el", "Ελληνικά"},
   {"be", "Беларуская"},
+  {"bg", "Български"},
```

Comment:
Successfully compiled whithout changing the constant (iOS only shows the error) :)

Why not to use old-school ```std::pair<std::string, std::string> kLanguageList[] = ...```

Reply:
1. tts_languages.py generates the wrong file without language names.
2. constexpr is better here